### ISSUE TITLE
Switch Javascript

### ISSUE THREAD

**Issue by haamza** [FreeCodeCamp/FreeCodeCamp#5781]
Challenge [Waypoint: Selecting from many options with Switch Statements](http://www.freecodecamp.com/challenges/waypoint-selecting-from-many-options-with-switch-statements#?solution=function%20myTest(val)%20%7B%0A%20%20var%20answer%20%3D%20%22%22%3B%0A%20%20%2F%2F%20Only%20change%20code%20below%20this%20line%0A%20%20switch%20(val)%20%7B%0A%20%20%20%20case%201%3A%0A%20%20%20%20%20%20console.log(%22alpha%22)%3B%0A%20%20%20%20%20%20break%3B%0A%20%20%20%20case%202%3A%0A%20%20%20%20%20%20console.log(%22beta%22)%3B%0A%20%20%20%20%20%20break%3B%0A%20%20%20%20case%203%3A%0A%20%20%20%20%20%20console.log(%22gamma%22)%3B%0A%20%20%20%20%20%20break%3B%0A%20%20%20%20case%204%3A%20%0A%20%20%20%20%20%20console.log(%22delta%22)%3B%0A%20%20%20%20%20%20break%3B%0A%20%20%20%20%20%20%0A%20%20%7D%0A%20%20%20%20%0A%20%20%2F%2F%20Only%20change%20code%20above%20this%20line%20%20%0A%20%20return%20answer%3B%20%20%0A%7D%0A%0A%2F%2F%20Change%20this%20value%20to%20test%0AmyTest(1)%3B%0A%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 5.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.73 Safari/537.36 OPR/34.0.2036.25</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript
function myTest(val) {
  var answer = "";
  // Only change code below this line
  switch (val) {
    case 1:
      console.log("alpha");
      break;
    case 2:
      console.log("beta");
      break;
    case 3:
      console.log("gamma");
      break;
    case 4: 
      console.log("delta");
      break;
      
  }
    
  // Only change code above this line  
  return answer;  
}

// Change this value to test
myTest(1);


```

**Comment by haamza** [comment]
Please someone tell me why its not functioning

**Comment by ltegman** [comment]
Issue does not describe a problem. Github issues are not for challenge help, please visit the [Help Chat](https://gitter.im/FreeCodeCamp/Help) for challenge help and then file an issue if they confirm that what you are experiencing is a site bug.

**Comment by ferraucchio** [comment]
same problem here can't figure out what i have to do

**Comment by theflametrooper** [comment]
@ferraucchio Please use the [help chat room](https://gitter.im/FreeCodeCamp/Help) for getting help with challenges. If you believe this challenge needs updated then please open a new issue.

**Comment by ferraucchio** [comment]
I just found the problem and solved it, you need to change the function name!

**Comment by theflametrooper** [comment]
@ferraucchio Great!

**Comment by VultureBetrayer** [comment]
The problem with this challenge is the explanation doesn't describe how the variable in the original function ties to the switch statement.  There is no psuedo code showing how the function ties in so there is no way for someone new to this statement to understand what the challenge is asking to do.  I only figured this out by trying every possible variable combination until I got it to work, but this did nothing to tell me WHY it worked, I had to google multiple sites until I learned how it works.

This challenge should have an explanation describing how the variable in the function is tied to the switch statement and how the return statement is also tied in.  This  may seem trivial to someone who knows what to look for, but to a beginner the entire challenge is murky and confusing.
